src/utils/handleError.ts

```diff
@@ -0,0 +1,18 @@
+import { AxiosError } from 'axios';
+import { createStandaloneToast } from '@chakra-ui/react';
+
+const { toast } = createStandaloneToast();
+
+export const handleError = (error: unknown, errorTitle: string): void => {
+  let errorMessage = 'An unexpected error occurred. Please try again.';
```

Comment:
please add 'Please try again' to all error messages (as we do not want users to leave our site!!!)